internal/usecase/client/pendo/pendo_client_dump.go

```diff
@@ -41,7 +42,7 @@ func (c *pendoClientDump) SetMetadata(ctx context.Context, kind pendo.Kind, grou
 // SendTrackEvent
 func (c *pendoClientDump) SendTrackEvent(ctx context.Context, track *pendo.TrackRequest) error {
 	logger := app_context.LogFromCtx(ctx)
-	logger.Debug("pendo SendTrackEvent called")
+	logger.Debug(fmt.Sprintf("pendo SendTrackEvent called: %s", track.Event))
```

Comment:
optional: use structured primitives for slog.

```suggestion
	logger.Debug("pendo SendTrackEvent called",
		slog.String("Event", track.Event),
	)
```